common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3898,6 +3899,14 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(3)
 
+  val WORKER_WRITER_GATHER_API_ENABLED: ConfigEntry[Boolean] =
+    buildConf("celeborn.worker.writer.gatherApi.enabled")
```

Comment:
I would call it `celeborn.worker.flusher.local.gatherAPI.enabled` and mark it as internal, user would not turn off it unless hit bugs.